ppl/src/main/antlr/OpenSearchPPLParser.g4

```diff
@@ -216,7 +216,10 @@ booleanExpression
 
 relevanceExpression
     : relevanceFunctionName LT_PRTHS
-        field=relevanceArgValue COMMA query=relevanceArgValue
+        field=relevanceField COMMA query=relevanceQuery
+        (COMMA relevanceArg)* RT_PRTHS
+    | relevanceFunctionNameEx LT_PRTHS
```

Comment:
Should we rename it? Like `relevanceFunctionType1Name` and `relevanceFunctionType2Name`.
Or maybe to make `relevanceExpressionType1` and `relevanceExpressionType2`.

Reply:
Please be explicit with your naming as possible.  Type1 and Type2 don't mean anything.  If there's an order associated with the types in the list (and no other way to differentiate the types) we can call them first and second or something similar.

Reply:
Let's change it to have separate rule for single-field relevance functions and a separate rule for multi-field relevance functions.

This way we can avoid the cumbersome `?:` in `AstExpressionBuilder`. Instead there will be one method to visit single field relevance functions and another to visit multi-field relevance functions.